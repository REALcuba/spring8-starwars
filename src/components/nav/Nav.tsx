import { Link } from "react-router-dom"
const Nav = () => {
  return (
    <nav className="bg-black flex justify-center border-y border-gray-400">
      
      <ul className=" flex text-gray-400 group/menu">
        <li className="border border-r-0 border-gray-400 px-4 py-2 text-sm hover:border-b-blue-600 hover:text-white" >
          <Link to='/'>HOME</Link>
        </li>
        <li className="border border-gray-400 px-4 py-2 text-sm hover:border-b-blue-600 hover:text-white">
          <Link to ='/main-page'><span>STARSHIPS</span></Link>
        </li>
      </ul>
    </nav>
  )
}

export default Nav
