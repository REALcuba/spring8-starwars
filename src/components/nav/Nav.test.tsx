import Nav from "./Nav";
import{describe,it, expect} from 'vitest';

describe('Nav',()=>{
    it('should be a function',()=>{
        expect(typeof Nav).toBe('function');
    });
    it('should render correctly',()=>{
        expect(Nav).toMatchSnapshot();
    });
})