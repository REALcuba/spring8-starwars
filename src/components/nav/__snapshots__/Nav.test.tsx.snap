// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Nav > should render correctly 1`] = `[Function]`;
